ext_emconf.php

```diff
@@ -18,10 +18,12 @@
     'version' => '4.1.0',
     'constraints' => [
         'depends' => [
-            'frontend' => '12.2.0-12.5.99',
-            'typo3' => '12.2.0-12.5.99'
+            'frontend' => '12.4.3-12.5.99',
+            'typo3' => '12.4.3-12.5.99'
+        ],
+        'conflicts' => [
+            'typo3' => ['12.0.0-12.4.2']
```

Comment:
do we need this conflict? isn't the lower boundary enough "'typo3' => '12.4.3-12.5.99'" ?

Reply:
Yeah, might be na overkill, I'll adjust it